redaxo/src/addons/metainfo/lang/de_de.lang

```diff
@@ -15,7 +15,7 @@ minfo_field_label_function = Funktion
 minfo_field_label_functions = Funktionen
 minfo_field_label_id = Id
 minfo_field_label_type = Feldtyp
-minfo_field_label_attributes = Feldattribute
+minfo_field_label_attributes = HTML-Attribute
```

Comment:
```
HTML/Feld-Attribute 
```

Reply:
`HTML-Attribute und Parameter`?

Reply:
"Parameter" heißt schon das Feld darüber (für die select-options etc.)

Reply:
Blöd gefragt: Gehören die Angaben nicht eigentlich auch genau dorthin?

Reply:
Eigentlich ja. Aber klick die Feldtypen mal durch und schau, was da so für Parameter (Syntax) möglich sind. Bei den Select/Radio/Checkbox wird es schwierig, das dort zu integrieren. Bei den Widgets hingegen haben wir da schon die Parameter-Schreibweise, da würde es gehen.

Dieses Feld hier gefällt mir so jedenfalls auch nicht mit der Mischung. Aber wir haben es aktuell nun mal so.

Reply:
Dann wäre ich dafür, es "HTML-Attribute" zu nennen und damit zu leben, dass da dann auch in 2 Fällen die Bezeichnung nicht treffend ist, weil sie eigentlich auch nur historisch so sind.

Reply:
Da fände ich die Variante "Feldattribute" plus Notiz unter dem Feld besser.